painters/normalization.py

```diff
@@ -0,0 +1,241 @@
+# Overwrite the Keras BatchNormalization class to create mode=2 for BatchNormalization layer.
```

Comment:
If this file is taken from a stack overflow answer / github issue / other web link, may be worth throwing that in the file documentation.